error.go

```diff
@@ -170,9 +170,9 @@ func (e *Error) Unwrap() error {
 // Format implements the Formatter interface.
 // Supported verbs:
 //
-// 		%s		simple message output
-// 		%v		same as %s
-// 		%+v		full output complete with a stack trace
+//	%s		simple message output
```

Comment:
Latest go reformats comments this way - can I leave it this way? Other contributors most likely will use modern go as well.